pkg/destroy/gcp/firewall.go

```diff
@@ -32,6 +32,13 @@ func (o *ClusterUninstaller) listFirewallsWithFilter(fields string, filter strin
 					key:      item.Name,
 					name:     item.Name,
 					typeName: "firewall",
+					quota: []quotaUsage{{
+						metric: &metric{
+							Service: serviceComputeEngineAPI,
+							Limit:   "firewalls",
```

Comment:
A lot of these types have a `kind` parameter, such as `compute#firewall`. this includes both the service and the item in its response. Not sure if it's worth trying to use these in most cases. I realize there are some special circumstances.

Reply:
I'm happy to factor this to do that, but wanted to make the least invasive change possible - what would you prefer?

Reply:
What you have is probably sufficient. I'd just make sure the limit strings match closely to some google defined string for the same resource.

Reply:
Oh - they match 100% to the quota API names. They do not match the `cloudResource.typeName` field